integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonMergerRDD.scala

```diff
@@ -456,6 +470,33 @@ class CarbonMergerRDD[K, V](
     result.toArray(new Array[Partition](result.size))
   }
 
+  private def getTaskNo(
+      split: CarbonInputSplit,
+      partitionTaskMap: util.Map[List[String], String]): String = {
+    if (carbonLoadModel.getCarbonDataLoadSchema.getCarbonTable.isStandardPartitionTable) {
```

Comment:
I remember their is a comment to rename `isStandardPartitionTable` to `isHivePartitionTable` in earlier PR, it was not fixed?

Reply:
yes, it is already changed but after rebase I did not fix compile issues. Now I fixed.